### PR TITLE
Add `site update` command

### PR DESCRIPTION
**Pull request by mjangda**
Triggers an `upgrade_wp` action for the given site and provides progress on container state throughout.

```
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
Triggering web server update/rebuild:
-- Site: example.com (#309)
-- Environment: production
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-

Successfully created wp upgrade host action for site 123 (action #123456)

## Container Status (updated 2017-01-11T19:33:38.406Z):
#10006 - example-comgo-vipco_web_0004 - running
#10007 - example-comgo-vipco_web_0005 - running

## wp-cli Status:
Action #123456: queued
Update complate 🎉🎉🎉
```

We wait for the `core update-db` host action to complete as the signal that we're done. The `Container Status` and `wp-cli Status` sections update throughout the process.

### CODE REVIEW

File: src/lib/site.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+// Ours
+const api = require( '../lib/api' );
+
+export function update( site ) {
+	return new Promise( ( resolve, reject ) => {
```
Comment (joshbetz) [inline src/lib/site.js]: We should add the minimum required node version to `package.json` and ensure all sandboxes are on the minimum version for Promises.

Comment (mjangda) [inline src/lib/site.js]: Sure. Looks like supports been in since `v0.12.1` (http://node.green/#Promise)

**Comment by ethitter** [comment]
Not sure if it's specific to this command, but there should be some validation applied to the site ID input. I typo'd a `=` onto one and it made the API query, which then errored.

File: src/lib/site.js (hunk 1 above)
Comment (mjangda) [inline src/lib/site.js]: Done in 9950e70

**Comment by mjangda** [comment]
There's one small bug where the status of `wp-cli` command can still show as "queued" (we don't do a final update before showing the error).

**Comment by mjangda** [comment]
> There's one small bug where the status of wp-cli command can still show as "queued"

Should be fixed in ddbf788

**Pull request merged**